### DISCUSSION TITLE
Issue AR Placement

### DISCUSSION THREAD

**Discussion by ghost** [google/model-viewer#3609] [Q&A]
Hi! 
I have a problem with my implementation, I put ar-placement="wall" on my element, but the element shows in any place, it ignores the ar placement,

You can check in these link: http://testglb.orgfree.com/

This is my code
`
<model-viewer src="https://modelviewer.dev/assets/boom_2_.glb" ar ar-placement="wall" ar-modes="webxr scene-viewer quick-look" alt="A 3D model of some wall art"></model-viewer>`

**Comment by elalish** [comment]
A screen recording would help a lot with understanding your situation. Keep in mind that ARCore has a tough time detecting walls, so you may have to move around quite a bit before it moves to object onto a real wall.

**Reply by ghost** [reply]
These are a few videos, a month ago I didn't have this problem

https://user-images.githubusercontent.com/49101408/181832311-28a531a8-cb5b-4967-aa90-cf96e6fd74f9.MOV

https://user-images.githubusercontent.com/49101408/181832478-2539ed06-a9a9-4307-8149-465332efacdb.MOV

**Reply by elalish** [reply]
Okay, so first off, you're first video is Scene Viewer and your second is Quick Look (both of which are separate from MV). You do specify WebXR mode first (our default, and the one that's actually part of our code base), so you should get it on Android. If you're not, it probably means you're not on HTTPS or put your side in an iframe that doesn't have the xr permission. You may want to check your JS console logs. 

As for why both Scene Viewer and Quick Look would suddenly work less well - I'd have to guess you're using a wall with fewer features or a different phone or something.

**Reply by Cloverclamp** [reply]
Hi guys, have you solve the issue of wall placement? I'm working to find the solution but nothing yet....
